acceptance/acceptance_test.go

```diff
@@ -14,6 +14,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build unit
```

Comment:
We probably don't want a build tag for the acceptance tests.

Reply:
:+1: